src/pages/shoppingList.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+
+const ShoppingList = () => {
```

Comment:
This way is fine, but for the final version, do it the same way in all functional components. Consistency 👍